src/gauges.rs

```diff
@@ -258,56 +258,57 @@ impl PrometheusGauges {
             &geolocations.len()
         );
 
-        let (uncached_ok, uncached_err): (Vec<_>, Vec<_>) =
-            futures::future::join_all(uncached.into_iter().map(|(contact, vote, _)| {
-                debug!(
-                    "Contacting Maxmind for: {:?}",
-                    get_rpc_contact_ip(&contact).unwrap()
-                );
-
-                self.client
-                    .get(format!(
-                        "{}/{}",
-                        MAXMIND_CITY_URI,
+        if let Some(maxmind) = exporter_config.maxmind.clone() {
```

Comment:
Should the check be made earlier such that `export_ip_addresses()` is never called if the config is missing? (i.e. in `main.rs`)